### PR TITLE
feature_header Header component added

### PR DESCRIPTION
**Pull request by dnnycortes**
- Header component added with it's template and styles

### CODE REVIEW

File: src/app/shared/components/header/header.component.html [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<header>
+    <nav>
+        <ul>
+            <li>
+                <a href="#">Home</a>
```
Comment (nike8806) [inline src/app/shared/components/header/header.component.html]: Pasar esto a un array

File: src/app/shared/components/header/header.component.scss [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+                display: flex;
+                flex-wrap: wrap;
+                justify-content: space-around;
+                list-style: none;
+                margin: 0;
+                padding: 0;
+
+                li {
+
+                    a{
```
Comment (nike8806) [inline src/app/shared/components/header/header.component.scss]: space

File: src/app/shared/components/header/header.component.ts [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import { Component, OnInit } from '@angular/core';
+
```
Comment (nike8806) [inline src/app/shared/components/header/header.component.ts]: remove space

File: src/app/shared/components/header/header.component.scss [hunk 4]
```diff
@@ -0,0 +1,33 @@
+:host {
+
```
Comment (nike8806) [inline src/app/shared/components/header/header.component.scss]: Add Media queries

**Comment by dnnycortes** [comment]
Comments fixed

**Pull request merged**